packages/replay-internal/src/replay.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable max-lines */ // TODO: We might want to split this file up
 import type { ReplayRecordingMode, Span } from '@sentry/core';
 import { getActiveSpan, getClient, getRootSpan, SEMANTIC_ATTRIBUTE_SENTRY_SOURCE, spanToJSON } from '@sentry/core';
-import { EventType, record } from '@sentry-internal/rrweb';
+import { EventType, record, utils as rrwebUtils } from '@sentry-internal/rrweb';
```

Comment:
I wonder if this can be tree-shaken, maybe I should just inline the function, see if that is smaller...